### PR TITLE
Fixed cs in manipulators

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

**Comment by fabpot** [comment]
Thank you @HeahDude.

File: Manipulator/KernelManipulator.php [hunk 1]
```diff
@@ -23,11 +23,6 @@ class KernelManipulator extends Manipulator
     protected $kernel;
     protected $reflected;
 
-    /**
-     * Constructor.
-     *
-     * @param KernelInterface $kernel A KernelInterface instance
```
Comment (OskarStark) [inline Manipulator/KernelManipulator.php]: why removing this?

Comment (HeahDude) [inline Manipulator/KernelManipulator.php]: Does not bring any value, the signature is explicit enough

Comment (OskarStark) [inline Manipulator/KernelManipulator.php]: makes no senso to me, but I'm fine with this.

I was just interested in this, thanks for the feedback

**Pull request merged**